src/pages/profile/Profile.jsx

```diff
@@ -5,76 +5,206 @@ import {
+                target="_blank"
+                rel="noreferrer"
+                className="flex flex-row gap-2 p-1"
+              >
+                <img src={Icon_portfolio2} alt="twitter" />
+                <p>Portfolio</p>
+              </a>
+              <a
+                href="https://google.com/"
+                target="_blank"
+                rel="noreferrer"
+                className="flex flex-row gap-2 p-1"
+              >
+                <img src={Icon_medium} alt="twitter" />
+                <p>Medium</p>
+              </a>
+            </div>
+            <div className="flex sm:flex-row  sm:gap-20 gap-5 flex-wrap">
+              <button
+                className="text-lg"
+                onClick={() =>
+                  setListModal((prev) => ({
+                    ...prev,
+                    showList: true,
+                    title: "Followers",
+                    list: userState.followers,
+                  }))
+                }
+              >
+                {userState.followers.length + " "}Followers
+              </button>
+              <button
+                className="text-lg"
+                onClick={() =>
+                  setListModal((prev) => ({
+                    ...prev,
+                    showList: true,
+                    title: "Following",
+                    list: userState.following,
+                  }))
+                }
+              >
+                {userState.following.length + " "}Following
+              </button>
+            </div>
+            {authState.user._id === userState._id && (
+              <button
+                className="bg-primary-color text-white px-2 py-1 rounded  hover:bg-primary-font-color w-max"
+                onClick={logoutUser}
+              >
+                Logout
+              </button>
+            )}
           </div>
-          <button
-            className="bg-primary-color text-white px-2 py-1 rounded  hover:bg-primary-font-color w-max"
-            onClick={logoutUser}
-          >
-            Logout
-          </button>
-        </div>
-      </main>
+          {userPosts.length === 0 ? (
+            <p>You do not have any posts yet</p>
+          ) : (
+            userPosts.map((post) => {
+              return (
+                <PostCard
+                  post={post}
+                  key={post._id}
+                  cardType={
+                    authState.user._id === userState._id ? "DELETE_CARD" : ""
+                  }
+                />
+              );
+            })
+          )}
+        </main>
+      )}
     </div>
   );
 };
 
 export { Profile };
+// {
+//   "_id": "f2eba8e4-d8ae-4510-9c3f-d7ad111f541f",
+//   "firstName": "Adarsh1",
+//   "lastName": "Balika1",
+//   "username": "adarshbalika",
+// }
+// "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiJmMmViYThlNC1kOGFlLTQ1MTAtOWMzZi1kN2FkMTExZjU0MWYiLCJ1c2VybmFtZSI6ImFkYXJzaGJhbGlrYSJ9.9L1GkdnqI3awI5E_4-zNz091L-bd9SJylC92a_DnpQ0"
```

Comment:
Can remove this commented code if not planning to use

Reply:
Removed commented code